src/profile-logic/import/instruments/index.js

```diff
@@ -0,0 +1,910 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+// @flow
+
```

Comment:
please add a comment explaining how you got the information to write this file, and some "as detailed as possible" details about the file format, so that we can follow along reading the code.